src/components/scenes/OtpErrorScene.tsx

```diff
@@ -196,7 +196,7 @@ export function OtpErrorScene(props: Props) {
       .finally(() => (inModal.current = false))
   })
 
-  const handleResetModal = useHandler(() => {
```

Comment:
Add an `await` to the `showResetModal` call down below. You can also get rid of the `.catch`, because the `await` will handle that.